### PR TITLE
Call [cssom-view]'s "getting the bounding box" algorithm directly

### LINKED ISSUE #55: Don't call public API methods; instead call their backing algorithms
A few places in the spec refer to public API methods:

> The DOMRect corresponding to the target’s getBoundingClientRect().
> 
> Let boundingClientRect be the value of target.getBoundingClientRect(). 
> 
> For each observer in node’s internal [[RegisteredIntersectionObservers]] slot, invoke unobserve() on observer, passing in node as target. 

This is not quite the right thing to do, since it implies that e.g. if I do `Element.prototype.getBoundingClientRect = () => { throw new Error("boo"); }`, you'll call that overriden function.

Instead, the correct thing to do is to refer to the backing concept or algorithm behind that method. For unobserve, you'd probably define something in the processing model like

"To **remove an intersection observer** _o_ from an Element _target_, perform the following steps: ..."

Then unobserve() algorithm would do "Remove the intersection observer _this_ from _target_," and instead of doing "invoke unobserve()," you would do a similar thing.

For getBoundingClientRect(), things are more annoying, since CSSOM View hasn't factored themselves appropriately. There you could do something like "Let boundingClientRect be a DOMRectReadonly obtained by the same algorithm as that of Element's getBoundingClientRect() performed on target."

### PR DESCRIPTION
**Pull request by miketaylr**
Closes #55

(And remove the stray package.json for the now-gone polyfill, oops).

***
<a href="https://pr-preview.s3.amazonaws.com/miketaylr/IntersectionObserver/pull/501.html" title="Last updated on Jul 6, 2022, 2:36 PM UTC (59d22ba)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/IntersectionObserver/501/eb168cb...miketaylr:59d22ba.html" title="Last updated on Jul 6, 2022, 2:36 PM UTC (59d22ba)">Diff</a>

### CODE REVIEW

**Comment by miketaylr** [comment]
PTAL @emilio @szager-chromium

File: polyfill/package.json [hunk 1]
```diff
@@ -1,23 +0,0 @@
-{
```
Comment (emilio) [inline polyfill/package.json:1]: Drive-by, but sure.

**Review by emilio — APPROVED** [review]
